packages/ui/src/Page/index.tsx

```diff
@@ -41,8 +43,14 @@ const Page: React.FC<Properties> = ({
     );
   }
 
+  if (className) {
+    className = "page" + ` ${className}`;
+  } else {
+    className = "page";
+  }
+
   return (
-    <div className="page">
+    <div className={className}>
```

Comment:
```suggestion
    <div className={`page ${className}`}>
```
With this, you can remove the if-else logic above.